src/data-visualisation/decluttering/decluttering.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: collection
+title: "Decluttering and pulling focus"
+description: "Guidance on the importance of decluttering and pulling focus for data visualisation"
+status: DRAFT
+tags: data
+order: 1
+collection_tag: dv-declutter
```

Comment:
Remove collection_tag & pagination settings as this is a single article not a collection of articles.